### PR TITLE
disable fatal warnings for a few projects

### PR DESCRIPTION
**Pull request by SethTisue**
because of scala/scala#5402

fyi @fommil @tpolecat

### CODE REVIEW

File: configs/community.dbuild [hunk 1]
```diff
@@ -588,9 +588,12 @@ build += {
     extra.run-tests: false
   }
 
+  // disable -Xfatal-warnings because of new unused warnings introduced
+  // by https://github.com/scala/scala/pull/5402
   ${vars.base} {
     name: "pcplod"
     uri:  ${vars.uris.pcplod-uri}
+    extra.commands: ${vars.default-commands} [ "removeScalacOptions -Xfatal-warnings" ]
```
Comment (adriaanm) [inline configs/community.dbuild:596]: is the error in pcplod going to be fixed by this? I think it's an error in its test suite, which is likely not affected by the compiler used to compile the project?

**Comment by adriaanm** [comment]
@SethTisue, Did you also mean to also do this for kind-projector? See also my comment about pcplod

**Comment by SethTisue** [comment]
@som-snytt's [kind-projector PR](https://github.com/non/kind-projector/pull/48) was already merged

as for pcplod, I'll see how the run goes and take appropriate action.

**Comment by adriaanm** [comment]
ah, 👍

**Comment by adriaanm** [comment]
I pro-actively filed https://github.com/ensime/pcplod/issues/20

File: configs/community.dbuild (hunk 1 above)
Comment (fommil) [reply configs/community.dbuild:596]: interesting, I should change the `scalacOptions` used in test to be definitive and not inherit from the build.

Comment (fommil) [reply configs/community.dbuild:596]: I'm travelling right now, so might be the weekend before I get to this.

**Pull request merged**